collect_app/src/test/java/org/odk/collect/android/backgroundwork/AutoSendTaskSpecTest.kt

```diff
@@ -47,15 +47,16 @@ class AutoSendTaskSpecTest {
     fun setup() {
         CollectHelpers.overrideAppDependencyModule(object : AppDependencyModule() {
             override fun providesInstanceAutoSender(
-                instanceAutoSendFetcher: InstanceAutoSendFetcher,
+                networkStateProvider: NetworkStateProvider?,
+                settingsProvider: SettingsProvider?,
                 context: Context?,
                 notifier: Notifier?,
                 analytics: Analytics?,
                 googleAccountsManager: GoogleAccountsManager?,
                 googleApiProvider: GoogleApiProvider?,
                 permissionsProvider: PermissionsProvider?,
                 instancesAppState: InstancesAppState?
-            ): InstanceAutoSender {
+            ): InstanceAutoSender? {
```

Comment:
I think this might have slipped in from an auto signature change?

Reply:
Right, fixed.